tests/integration/functional/cli/test_stack_recipes.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import logging
+import os
+from pathlib import Path
+
+import pytest
+from click.testing import CliRunner
+
+from zenml.cli.cli import cli
+from zenml.cli.stack_recipes import STACK_RECIPES_REPO_DIR
+from zenml.utils import io_utils
+
+SERVER_START_STOP_TIMEOUT = 30
+
+
+@pytest.fixture(scope="module")
+def cli_runner():
+    """Fixture to get a CliRunner instance."""
+    return CliRunner()
```

Comment:
Maybe this could live inside `conftest.py` and just reused from there?

Reply:
Yes that was the approach I was using earlier but looking at how the other cli tests are written, I used this design to stay consistent.